hasura/migrations/default/1703174336818_create_role_and_membership_tables/up.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE role (
+  value text PRIMARY KEY,
+  comment text
+);
+
+INSERT INTO role (value, comment) VALUES
+  ('OWNER', 'Owner of the team'),
+  ('ADMIN', 'Users with the privilege to manage other users'),
+  ('MEMBER', 'Member user');
+
+CREATE TABLE "public"."memberships" (
```

Comment:
my mistake, realizing we use singular for all table names,
```suggestion
CREATE TABLE "public"."membership" (
```